src/handlers/scheduledEventHandler.ts

```diff
@@ -1,24 +1,26 @@
 import { env } from "../typeDefinitions/default.types";
 import { taskOverDueDiscordMembers } from "../utils/taskOverDueDiscordMembers";
-import * as error from "../constants/responses";
-import { getDiscordIds } from "../utils/getDiscordIds";
 import config from "../../config/config";
 import { SUPER_USER_ONE, SUPER_USER_TWO } from "../constants/variables";
 
 export async function send(env: env): Promise<void> {
   try {
-    const assigneeIds: string[] | string = await taskOverDueDiscordMembers();
+    const discordIds: string[] = await taskOverDueDiscordMembers();
 
-    //A user might have more than one task which are running red
-    //so to mention them just once, we are using Set to filter out
-    const discordIds: string[] | string = await getDiscordIds(assigneeIds);
-    const uniqueDiscordIds = [...new Set(discordIds)];
+    const superUsers = [SUPER_USER_ONE, SUPER_USER_TWO];
```

Comment:
NIT: Can we make this dynamic, today we are excluding 2 superusers, later if there are more then this will give wrong output (Can ignore if not relevent)

Reply:
I think we can make it dynamic by getting member list by role id and then exclude but it willl take time to get member list and filter